landing-pages/site/content/en/blog/airflow-1.10.8-1.10.9.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Apache Airflow 1.10.8 & 1.10.9"
+linkTitle: "Apache Airflow 1.10.8 & 1.10.9"
+author: "Kaxil Naik"
+twitter: "kaxil"
+github: "kaxil"
+linkedin: "kaxil"
+description: "We are happy to present the new 1.10.8 and 1.10.9 releases of Apache Airflow."
+tags: ["release"]
+date: 2020-02-23
+---
+
+We are happy to present the new 1.10.8 and 1.10.9 releases of Apache Airflow.
+
+Airflow 1.10.8 contains 160 commits since 1.10.7 and includes 4 new features, 42 improvements, 36 bug fixes, and several doc changes.
+
+We released 1.10.9 on the same day as one of the Flask dependencies (Werkzeug) released 1.0 which broke Airflow 1.10.8.
+
+**Details**:
+* **PyPI**: [https://pypi.org/project/apache-airflow/1.10.9/](https://pypi.org/project/apache-airflow/1.10.9/)
+* **Docs**: [https://airflow.apache.org/docs/1.10.9/](https://airflow.apache.org/docs/1.10.9/)
+* **Changelog (1.10.8)**: [http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07](http://airflow.apache.org/docs/1.10.8/changelog.html#airflow-1-10-8-2020-01-07)
+* **Changelog (1.10.9)**: [http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10](http://airflow.apache.org/docs/1.10.9/changelog.html#airflow-1-10-9-2020-02-10)
+
+Some of the noteworthy new features (user-facing) are:
+
+- [Add tags to DAGs and use it for filtering in the UI](https://github.com/apache/airflow/pull/6489)
+- [New Executor: DebugExecutor for Local debugging from your IDE](http://airflow.apache.org/docs/1.10.9/executor/debug.html)
+- [Allow passing conf in "Add DAG Run" view](https://github.com/apache/airflow/pull/7281)
+- [Allow dags to run for future execution dates](https://github.com/apache/airflow/pull/7038)
```

Comment:
This is specifically only when schedule_interval=None isn't it?

Reply:
Yes

Reply:
We should say that here then

Reply:
Updated